source/element-parameters.md

```diff
@@ -66,387 +69,68 @@ Element:
 ```
 
 %---------------------------------------------------------------------------------------------------
-(s:ac.kicker.params)=
-## ACKickerP:  AC-Kicker Parameters
+## Dependent and User Set parameters.
 
-In Construction...
+Some parameters are set in the lattice file. These parameters are called "User settable". 
```

Comment:
I'm not sure about the word "settable".  I suggest calling these "User set" parameters, using the same phrase that appears in the title of this section.

Reply:
The problem with "User set" is that this implies that the User has set the parameter. As opposed to "User settable" which implies that the User **may** set the parameter.